pkgs/applications/misc/bashSnippets/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, fetchFromGitHub
+, curl, netcat, mpv, python, bind, iproute, bc, gitMinimal, ... }:
+let
+  version = "1.12.0";
+in
+stdenv.mkDerivation {
+  name = "bashSnippets-${version}";
+
+  src = fetchFromGitHub {
+    owner = "alexanderepstein";
+    repo = "Bash-Snippets";
+    rev = "v${version}";
+    sha256 = "0kx2a8z3jbmmardw9z8fpghbw5mrbz4knb3wdihq35iarcbrddrg";
+  };
+
+  propagatedBuildInputs = [
```

Comment:
I thought about that, but:
- The dependencies are pretty minimal, the only thing I could image not being installed already is mvp
- This is the perfect opportunity for Nix to show that it can actually manage dependencies well and that you can expect the commands you just installed to work

I also thought about enabling the user to install each snippet separately, but I think this would be more work than it's worth.

What do you think of just adding mvp as an optional component?